### PR TITLE
LPS-171909 inputTrigger was altered so it would affect the picklists in forms as well

### PR DESCRIPTION
**Pull request by ericthmoritsuka**
Related Issues
https://issues.liferay.com/browse/LPS-171909

### CODE REVIEW

**Comment by ericthmoritsuka** [comment]
ci:test:sf

**Comment by ericthmoritsuka** [comment]
ci:test:sf

**Comment by ericthmoritsuka** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
Hey @cgmonte,
Could you review this PR please? :smile:

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js [hunk 1]
```diff
@@ -347,9 +347,10 @@ const Select = ({
 		}
 	};
 
-	const inputTrigger = document.querySelector(
+	const inputTrigger = document.querySelectorAll(
 		'.lfr__ddm-select-input-trigger'
-	);
+	)[document.querySelectorAll('.lfr__ddm-select-input-trigger').length - 1];
+
```
Comment (cgmonte) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js:353]: Hey, @ericthmoritsuka! Thanks for working on this issue! I see you are trying to select a specific element with a ".lfr__ddm-select-input-trigger " class, but doing that by getting the lest element in the node list is not the safest way of doing it. We may have cases when this element is not the last one!

I suggest you use the child combinator selector and select the parent by attribute value, as bellow: 

```
const inputTrigger = document.querySelector(
	'[data-field-reference|="predefinedValue"] > .lfr__ddm-select-input-trigger'
);
```

**Review by cgmonte — CHANGES_REQUESTED** [review]
Hi, @ericthmoritsuka! Thanks for working on this issue! Please see my comment suggesting a change a let me know what you think about it.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js (hunk 1 above)
Comment (ericthmoritsuka) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js:353]: Hi Carlos, thanks for the review! I thought this fix would not be ideal and I was wondering what could be done to make it work the right way. Thank you for the suggestion. I will work on it and get back to you.

**Comment by marcelabc** [comment]
Hi @ericthmoritsuka, 
Since there are changes to be made, I will close this ticket and change the Jira Status to In progress, okay? 
Let us know if you need help!

**Pull request closed without merging**